CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributors guide
+We are happy You are considering contributing to jbehave-support! 😃
+
+Before you start, please read this short guide, so You don`t get lost. If You have any questions, do not hesitate to ask either on Gitter or through GitHub issues.
+
+## Dev environment
+This project is written in Java 8 using the latest version of IntelliJ IDEA as an IDE. These are the plugins we use:
```

Comment:
Just a thought - might be good mentioning that while we are primarily targeting java 8 we are also aiming at java 11 compatibility (we run tests again both in CI)